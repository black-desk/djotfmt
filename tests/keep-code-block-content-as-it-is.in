{% vim:set ft=djot: %}

`djotfmt` keep all content in code block as it is.

* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *

```
Some   random
text with  multiple spaces    and


empty



lines    in



between.
```
