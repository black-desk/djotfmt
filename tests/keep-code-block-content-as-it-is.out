keep-code-block-content-as-it-is.in